### PR TITLE
new requests

### PR DESCRIPTION
**Pull request by Alex-barrett61**
hey

### CODE REVIEW

File: src/controllers/comment.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+   */
+  async get(id) {
+    const comment = await this.model.Get(id);
+
+    if (!comment) {
+      console.log('comment not found');
+      return {};
+    }
+    return comment;
+  }
```
Comment (ryan-barrett) [inline src/controllers/comment.js:20]: put a space here

File: src/controllers/comment.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  /**
+   *
+   * @param {string} body
+   * @returns {Promise<void>}
```
Comment (ryan-barrett) [inline src/controllers/comment.js:24]: this doesn't return void it returns an id right?

File: src/controllers/post.js [hunk 3]
```diff
@@ -23,10 +34,12 @@ class PostController extends Controller {
    *
    * @param {string} title
    * @param {string} body
+   * @param {string} userId
    * @returns {Promise<void>}
```
Comment (ryan-barrett) [inline src/controllers/post.js:38]: this also doesn't return void

File: src/controllers/post.js [hunk 4]
```diff
@@ -41,13 +54,14 @@ class PostController extends Controller {
...
    * @param {string} title
+   * @param {string} userId
    * @param {string} body
    * @param {string} id
    * @returns {Promise<void>}
    */
 
-  async update(title, body,id) {
-    const update = new this.model(title, body, 'test', 'test');
+  async update(title, body,userId,id) {
```
Comment (ryan-barrett) [inline src/controllers/post.js:63]: fix the linting

File: src/controllers/user.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
+const Controller = require('./controller');
+const UserModel = require('../models/user');
+
+class UserController extends Controller {
+  constructor() {
+    super(UserModel);
+  }
+  /**
```
Comment (ryan-barrett) [inline src/controllers/user.js:8]: need a space here

File: src/controllers/user.js [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    return user;
+  }
+
+  /**
+   *
+   * @param {string} email
+   * @param {string} name
+   * @param {string} password
+   * @param {string} phone
+   * @returns {Promise<void>}
```
Comment (ryan-barrett) [inline src/controllers/user.js:40]: this doesn't return void

File: src/controllers/user.js [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+    return this.model.Delete(id);
+  }
+  /**
+   *
+   * @param {string} id
+   * @param {string} email
+   * @param {string} name
+   * @param {string} password
+   * @param {string} phone
+   * @returns {Promise<void>}
```
Comment (ryan-barrett) [inline src/controllers/user.js:62]: doesn't return void

File: src/models/post.js [hunk 8]
```diff
@@ -52,7 +70,7 @@ class PostModel extends Model {
   static async Insert(post) {
     try {
       const { id, title, body, userId, photoUrl } = post;
-
+    console.log('got here',id)
```
Comment (ryan-barrett) [inline src/models/post.js:73]: remove this

File: src/models/user.js [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+        [id, name, email, password, email]
+      );
+      return { id };
+    }
+    catch (error) {
+      console.log('Error creating user', error);
+      return {};
+    }
+  }
+  /**
```
Comment (ryan-barrett) [inline src/models/user.js:80]: space here

File: src/models/user.js [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+        [id]
+      );
+      return rows[0];
+    }
+    catch (error) {
+      console.log('Error getting comment', error);
+      return {};
+    }
+  }
+  /**
```
Comment (ryan-barrett) [inline src/models/user.js:59]: space here

File: src/models/user.js [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+    console.log(id);
+    try {
+      await this.postgres.query('DELETE FROM users WHERE id = $1 RETURNING *;', [id]);
+      return true;
+    }
+    catch (error) {
+      console.log('Error deleting users', error);
+      return false;
+    }
+  }/**
```
Comment (ryan-barrett) [inline src/models/user.js:95]: space here

File: src/services/comment.js [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+   * @param {string} id - the id of a single post
+   * @returns {Promise<object>}
+   */
+  async get(id) {
+    console.log('fetching comment by id', id);
+    const comment = await this.controller.get(id);
+    console.log('returning comment', comment);
+    return comment;
+  }
+  async create(data) {
```
Comment (ryan-barrett) [inline src/services/comment.js:20]: put spaces between these functions

File: src/services/post.js [hunk 13]
```diff
@@ -6,6 +6,17 @@ class PostService extends Service {
     super(...args);
     this.controller = new PostController();
   }
+  /**
```
Comment (ryan-barrett) [inline src/services/post.js:9]: space

File: src/services/post.js [hunk 14]
```diff
@@ -20,8 +31,9 @@ class PostService extends Service {
   }
 
   async create(data) {
-    const { title, body } = data;
-    return this.controller.create(title, body);
+    console.log('got here')
```
Comment (ryan-barrett) [inline src/services/post.js:34]: remove the got here

File: src/services/user.js [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+    const user = await this.controller.getPosts(id);
+    console.log('returning user', user);
+    return id;
+  }
+  async get(id) {
+    console.log('fetching user by id', id);
+    const user = await this.controller.get(id);
+    console.log('returning user', user);
+    return user;
+  }
```
Comment (ryan-barrett) [inline src/services/user.js:26]: spaces between the functions

**Review by ryan-barrett — CHANGES_REQUESTED** [review]

File: src/controllers/comment.js (hunk 2 above)
Comment (Alex-barrett61) [reply src/controllers/comment.js:24]: If this is a question then no it does not return void I dont think anyways

Comment (ryan-barrett) [reply src/controllers/comment.js:24]: if the return isn't right you should fix or remove it

Comment (Alex-barrett61) [reply src/controllers/comment.js:24]: everything returns an Id

**Review by ryan-barrett — APPROVED** [review]

**Pull request merged**